README.rst

```diff
@@ -43,6 +43,11 @@ ELASTICSEARCH_AWS_{ACCESS_KEY_ID,SECRET_ACCESS_KEY,REGION}
   request signing with the given credentials. This allows making requests to an
   AWS Elasticsearch domain as an IAM user.
 
+HYPOTHESIS_AUTHORITY
```

Comment:
The main reason I went with checking for "not the first party authority" is because I can't think of a situation in which Via could reasonably work with third party authorities as long as we keep Via's privacy/security features such as not stripping cookie-related headers.

Direct linking to annotations in Canvas will definitely require additional work because the annotated page is not the top-level frame in the document but is hosted inside an iframe. Also, IIRC the plan is that Canvas will always use Via directly rather than going through Bouncer.

Lastly, we also have a few inbound requests from others interested in third party authorities and I would like to lower the barrier to potential users trying out the feature by not requiring a config change to Bouncer to enable new third-party authorities to work.

Reply:
Ok. Yup, that all sounds fair to me; thanks for the explanation. If we do find out that this causes some kind of Canvas-related problems, this doesn’t seem like it’ll be hard to adapt later.